### PR TITLE
Fix/url room

### PR DESCRIPTION
**Pull request by OkoloJohnbosco**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

# Changes proposed

## What were you told to do?
To use the organisation/workspace id stored in localstorage to make api call instead of the short_id in the url

## What did you do?
Fixed localstorage issue, someone was accessing localstorage without JSON.parse() 😄 
Replaced url location pathname with the workspace/organization id stored in the local storage
# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos
image

image

### CODE REVIEW

**Comment by OkoloJohnbosco** [comment]
The error 500 is from the server i talked with @zxenonx

File: packages/main/src/components/protected/sidebar/components/ModalComponent.jsx [hunk 1]
```diff
@@ -14,19 +14,25 @@ const ModalComponent = ({ workSpace, isOpen, toggleOpenInvite }) => {
...
+              Authorization: `Bearer ${userData.token}`
+            }
+          }
+        )
         .then(res => {
           setOrgLogoUrl(res.data.data.logo_url ? res.data.data.logo_url : "");
         })
         .catch(err => {
-          console.error(err);
+          console.error(err, "from modal component");
```
Comment (billmal071) [inline packages/main/src/components/protected/sidebar/components/ModalComponent.jsx:35]: Use a toast notification for alerts and errors.

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 2]
```diff
@@ -77,6 +77,7 @@ export default function Index() {
 
   const fetchUserWorkspacesResponse = async () => {
     let userData = JSON.parse(sessionStorage.getItem("user"));
+    console.log(userData, "userData");
```
Comment (billmal071) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:81]: remove.

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 3]
```diff
@@ -86,6 +87,10 @@ export default function Index() {
           }
         }
       );
+      console.log(
+        `https://api.zuri.chat/users/${userData.email}/organizations`,
+        "omo"
```
Comment (billmal071) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:93]: remove.

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 4]
```diff
@@ -105,8 +110,10 @@ export default function Index() {
 
   useEffect(() => {
     window.dispatchEvent(new Event("zuri-plugin-load"));
-    match.isExact &&
+    match?.isExact &&
       history.replace(`/workspace/${short_id}/plugin-chat/all-dms`);
+
+    console.log(short_id, "short_id");
```
Comment (billmal071) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:117]: remove.

**Comment by OkoloJohnbosco** [comment]
@billmal071  TheShinobi it is I

**Review by billmal071 — CHANGES_REQUESTED** [review]

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 5]
```diff
@@ -78,6 +78,7 @@ export default function Index() {
 
   const fetchUserWorkspacesResponse = async () => {
     let userData = JSON.parse(sessionStorage.getItem("user"));
+    console.log(userData, "userData");
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:81]: Remove this console.log

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 6]
```diff
@@ -106,8 +111,10 @@ export default function Index() {
 
   useEffect(() => {
     window.dispatchEvent(new Event("zuri-plugin-load"));
-    match.isExact &&
+    match?.isExact &&
       history.replace(`/workspace/${short_id}/plugin-chat/all-dms`);
+
+    console.log(short_id, "short_id");
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:117]: Remove this console.log

File: packages/main/src/pages/protected/workspace/Workspace.jsx [hunk 7]
```diff
@@ -87,6 +88,10 @@ export default function Index() {
           }
         }
       );
+      console.log(
+        `https://api.zuri.chat/users/${userData.email}/organizations`,
+        "omo"
+      );
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/workspace/Workspace.jsx:94]: What was happening here?🤣. Remove this console.log too

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Comment by billmal071** [comment]
@OkoloJohnbosco

File: packages/main/src/components/protected/sidebar/components/ModalComponent.jsx (hunk 1 above)
Comment (OkoloJohnbosco) [reply packages/main/src/components/protected/sidebar/components/ModalComponent.jsx]: Omo, Ok boss

File: packages/main/src/pages/protected/workspace/Workspace.jsx (hunk 7 above)
Comment (OkoloJohnbosco) [reply packages/main/src/pages/protected/workspace/Workspace.jsx:94]: lol

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**